package.json

```diff
@@ -3,18 +3,26 @@
   "version": "0.1.1",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
+    "@fortawesome/free-solid-svg-icons": "^5.13.0",
+    "@fortawesome/react-fontawesome": "^0.1.9",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.1",
     "styled-components": "^5.1.0"
   },
   "scripts": {
     "start": "react-scripts start",
```

Comment:
Maybe add a headless script to prevent the broswer from opening when running cypress tests?
`"start:headless": "broswer=NONE npm start"`

Reply:
Sorry, could not get it running yet. Tried out different approaches, .env files, set "REACT_APP_BROWSER... " inside the script etc. None worked. Maybe you have an idea?

Reply:
Not at the moment.. -> another ticket :)

Reply:
Added to the backlog!